cluster/engine.go

```diff
@@ -425,7 +425,11 @@ func (e *Engine) Create(config *ContainerConfig, name string, pullImage bool) (*
 	e.RLock()
 	defer e.RUnlock()
 
-	return e.containers[id], nil
+	justCreated := e.containers[id]
```

Comment:
Sorry for being pedantic but why not `container` or `c`? Semantically speaking, everything on the left side of `:=` was *just created* :)